test-gas-costs/gasCosts.js

```diff
@@ -214,7 +214,12 @@ contract("All", function(accounts) {
       await advanceMiningCycleNoContest({ colonyNetwork, test: this });
       await advanceMiningCycleNoContest({ colonyNetwork, test: this, minerAddress: STAKER1 });
 
-      const goodClient = new ReputationMinerTestWrapper({ loader: contractLoader, minerAddress: STAKER1, realProviderPort: REAL_PROVIDER_PORT });
+      const goodClient = new ReputationMinerTestWrapper({
+        loader: contractLoader,
+        minerAddress: STAKER1,
+        realProviderPort: REAL_PROVIDER_PORT,
+        useJsTree: true
+      });
       const badClient = new MaliciousReputationMinerExtraRep(
         { loader: contractLoader, minerAddress: STAKER2, realProviderPort: REAL_PROVIDER_PORT },
```

Comment:
Shouldn't the bad client get to use it too? 😄

Reply:
Yes of course.. I'll do that